webextensions/api/windows.json

```diff
@@ -732,7 +732,7 @@
               "__compat": {
                 "support": {
                   "chrome": {
-                    "version_added": "≤61"
+                    "version_added": "12"
```

Comment:
✅ https://github.com/chromium/chromium/commit/d4db6c702d284543ba49b63eee555bf864becb09 in [12.0.718.0](https://github.com/chromium/chromium/releases/tag/12.0.718.0) before 12.0.742.